source/documentation/deploying_services/mysql.md

```diff
@@ -232,6 +232,35 @@ To upgrade, you must set up a new service and migrate your app data.
 
 You cannot currently downgrade your service plan.
 
+### Reboot a MySQL service instance
+
+You can [reboot](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_RebootInstance.html) [external link] your MySQL service instance to:
+
+- try to fix a problem with your service instance
+- test how your app behaves during a service instance failure
+
+To reboot your service instance, you must have [space developer](/orgs_spaces_users.html#space-developer) permissions in the [space](/orgs_spaces_users.html#spaces) that hosts your service instance.
```

Comment:
permissions -> permission

Reply:
We talked about this and `role` or `permissions` are both correct, but not `permission`.